### PR TITLE
[FR] Update  `Community/osu! Discord server`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/osu!_Discord_server/fr.md [hunk 1]
```diff
@@ -2,37 +2,62 @@
...
+  - osu!dev
 ---
 
 # Serveur Discord osu!dev
 
-**osu!dev** est le serveur [Discord](https://discordapp.com) officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
+Le serveur Discord **osu!** anciennement appelé **osu!dev**, est le serveur [Discord](https://discordapp.com) officiel pour le développement. Le serveur fonctionne comme un centre de discussion et de travail sur sur les projets open source et communautaires d'osu!. C'est l'endroit principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
+
 
 Le lien d'invitation permanent est <https://discord.gg/ppy>.
```
Comment (Sitoria) [inline wiki/Community/osu!_Discord_server/fr.md:13]: need blank line
```suggestion
Le lien d'invitation permanent est <https://discord.gg/ppy>.

```

**Comment by Sitoria** [comment]
topic name will be `Community/osu!_Discord_Server` cuz `osu!dev discord server` repo is renamed to `osu! Discord Server`

File: wiki/Community/osu!_Discord_server/fr.md [hunk 2]
```diff
@@ -2,37 +2,62 @@
...
-outdated_translation: true
-outdated_since: 7ca332517b46ef83f80a09c1b9082b437bacd39a
+  - osu!dev
 ---
 
 # Serveur Discord osu!dev
 
-**osu!dev** est le serveur [Discord](https://discordapp.com) officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
+Le serveur Discord **osu!** anciennement appelé **osu!dev**, est le serveur [Discord](https://discordapp.com) officiel pour le développement. Le serveur fonctionne comme un centre de discussion et de travail sur sur les projets open source et communautaires d'osu!. C'est l'endroit principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
+
```
Comment (Sitoria) [inline wiki/Community/osu!_Discord_server/fr.md:12]:
```suggestion
```
seems got error cuz 2 blank line appears - delete one?

File: wiki/Community/osu!_Discord_server/fr.md [hunk 3]
```diff
@@ -2,37 +2,62 @@
 tags:
   - development
   - développement
-outdated_translation: true
-outdated_since: 7ca332517b46ef83f80a09c1b9082b437bacd39a
+  - osu!dev
 ---
 
 # Serveur Discord osu!dev
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:8]:
```suggestion
# Serveur Discord osu!
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 4]
```diff
@@ -2,37 +2,62 @@
...
 
-**osu!dev** est le serveur [Discord](https://discordapp.com) officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
+Le serveur Discord **osu!** anciennement appelé **osu!dev**, est le serveur [Discord](https://discordapp.com) officiel pour le développement. Le serveur fonctionne comme un centre de discussion et de travail sur sur les projets open source et communautaires d'osu!. C'est l'endroit principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:10]:
```suggestion
Le serveur Discord **osu!**, anciennement appelé **osu!dev**, est le serveur [Discord](https://discordapp.com) officiel pour le développement. Le serveur fonctionne comme un centre de discussion et de travail sur sur les projets open source et communautaires d'osu!. C'est l'endroit principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 5]
```diff
@@ -2,37 +2,62 @@
...
-## Vérification
+## Salons
+
+Les nouveaux venus sur le serveur sont encouragés à se présenter dans le salon `#introductions`, mais ce n'est pas une obligation pour interagir avec le reste du serveur.
+
+### Général
+
+| Salon | Description |
+| :-- | :-- |
+| `#welcome` | Information serveur |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:23]:
```suggestion
| `#welcome` | Informations sur le serveur |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 6]
```diff
@@ -2,37 +2,62 @@
...
+
+### Client / Web
 
-Par défaut, seuls les utilisateurs vérifiés ont l'autorisation de discuter dans le reste du serveur Discord. Si un utilisateur non-vérifié souhaite le devenir, celui-ci doit poster un message dans le canal `#arrival` du serveur Discord accompagné de ses motivations et intentions. Une fois ceci fait, un membre de l'équipe d'osu! s'occupera de vérifier l'utilisateur. *Prenez note que la vérification n'est pas immédiate et qu'elle peut prendre un certain temps.*
+| Salon | Description |
+| :-- | :-- |
+| `#osu` | [osu!(lazer)](/wiki/Client/Release_stream/Lazer) |
+| `#osu-framework` | [osu!framework](https://github.com/ppy/osu-framework) |
+| `#osu-web` | [osu!web](https://github.com/ppy/osu-web) |
+| `#github-feed` | Flux d'événements pour divers [ppy GitHub repositories](https://github.com/ppy) |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:34]:
```suggestion
| `#github-feed` | Flux d'événements pour divers [repositories GitHub ppy](https://github.com/ppy) |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 7]
```diff
@@ -2,37 +2,62 @@
...
+| `Meetings` (Salon de conférence) | [réunions communautaire d'osu!](/wiki/Community/osu!_community_meetings) et réunions sur l'état du développement |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:35]:
```suggestion
| `Meetings` (Salon de conférence) | [osu! community meetings](/wiki/Community/osu!_community_meetings) et réunions sur l'état du développement |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 8]
```diff
@@ -2,37 +2,62 @@
...
 
-## Projets
+### Diffcalc
 
-Chaque partie du développement possède son propre canal de discussion :
+| Salon | Description |
+| :-- | :-- |
+| `#difficulty-osu` | Calcul du [star rating](/wiki/Beatmap/Star_rating) et des [points de performance](/wiki/Performance_points) pour [osu!](/wiki/Game_mode/osu!) |
+| `#difficulty-taiko` | Calcul du star rating et des points de performances pour [osu!taiko](/wiki/Game_mode/osu!taiko)  |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:42]:
```suggestion
| `#difficulty-taiko` | Calcul du star rating et des points de performance pour [osu!taiko](/wiki/Game_mode/osu!taiko)  |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 9]
```diff
@@ -2,37 +2,62 @@
...
+| `#difficulty-catch` | Calcul du star rating et des points de performances pour [osu!catch](/wiki/Game_mode/osu!catch) |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:43]:
```suggestion
| `#difficulty-catch` | Calcul du star rating et des points de performance pour [osu!catch](/wiki/Game_mode/osu!catch) |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 10]
```diff
@@ -2,37 +2,62 @@
...
+| `#difficulty-mania` | Calcul du star rating et des points de performances pour [osu!mania](/wiki/Game_mode/osu!mania) |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:44]:
```suggestion
| `#difficulty-mania` | Calcul du star rating et des points de performance pour [osu!mania](/wiki/Game_mode/osu!mania) |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 11]
```diff
@@ -2,37 +2,62 @@
...
 
-| Projet | Canal |
+### Communauté
+
+| Salon | Description |
+| :-- | :-- |
+| `#featured-artists` | création de beatmaps, de projets ou de playlists des [Featured Artist](/wiki/People/Featured_Artists) |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:50]:
```suggestion
| `#featured-artists` | Création de beatmaps, de projets ou de playlists des [Featured Artists](/wiki/People/Featured_Artists) |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 12]
```diff
@@ -2,37 +2,62 @@
...
+| `#mappers-guild` | La [Mappers' Guild](/wiki/Community/Mappers_Guild) |
+| `#osu-spotlights` | [Beatmap Spotlights](/wiki/Beatmap_Spotlights) |
+| `#osu-wiki` | [osu! wiki](/wiki/osu!_wiki) |
+| `#osu-news` | [osu!monthly](/wiki/Community/osu!monthly) et la couverture de l'actualité générale |
+| `#osu-loved` | [Project Loved](/wiki/Community/Project_Loved) |
+| `#tournaments` | [Tournaments](/wiki/Tournaments) and [official tournament support](/wiki/Tournaments/Official_support) |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:56]:
```suggestion
| `#tournaments` | [Tournois](/wiki/Tournaments) et le [support officiel aux tournois](/wiki/Tournaments/Official_support) |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Community/osu!_Discord_server/fr.md [hunk 13]
```diff
@@ -2,37 +2,61 @@
...
 
-## Projets
+### Général
 
-Chaque partie du développement possède son propre canal de discussion :
+| Salon | Description |
+| :-- | :-- |
+| `#welcome` | Informations sur le serveur |
+| `#introduction` | Présentation de soi |
+| `#general` | Discussion générale et dépannage liés à l'osu! |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:24]:
```suggestion
| `#general` | Discussion générale et dépannage liés à osu! |
```

File: wiki/Community/osu!_Discord_server/fr.md [hunk 14]
```diff
@@ -2,37 +2,61 @@
...
-outdated_translation: true
-outdated_since: 7ca332517b46ef83f80a09c1b9082b437bacd39a
+  - osu!dev
 ---
 
-# Serveur Discord osu!dev
+# Serveur Discord osu!
 
-**osu!dev** est le serveur [Discord](https://discordapp.com) officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
+Le serveur Discord **osu!**, anciennement appelé **osu!dev**, est le serveur [Discord](https://discordapp.com) officiel pour le développement. Le serveur fonctionne comme un centre de discussion et de travail sur sur les projets open source et communautaires d'osu!. C'est l'endroit principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Discord_server/fr.md:10]:
```suggestion
Le serveur Discord **osu!**, anciennement appelé **osu!dev**, est le serveur [Discord](https://discordapp.com) officiel pour le développement. Le serveur fonctionne comme un centre de discussion et de travail sur les projets open source et communautaires d'osu!. C'est l'endroit principal pour contribuer au développement d'osu! et entrer en contact avec [l'équipe d'osu!](/wiki/People/osu!_team).
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Pull request merged**